content-services/latest/develop/extension-inspector.md

```diff
@@ -2,46 +2,50 @@
 title: Extension Inspector
 ---
 
-Alfresco Extension Inspector allows you to analyse Alfresco extensions and to compare them against a particular Alfresco Content Service version. For example, it helps you to understand how customizations and extensions are affected with a newer version of Alfresco, checking for compliance with our best practices and providing recommendations for upgrade impact and safety.
+Alfresco Extension Inspector allows you to analyze Alfresco extensions and to compare them against a particular Alfresco Content Services version. For example, it helps you to understand how customizations and extensions are affected with a newer version of Content Services, checking for compliance with our best practices and providing recommendations for upgrade impact and safety. It works with Alfresco Community Edition and Alfresco Content Services 5.2.7, 6.2.2, and 7.0 releases.
```

Comment:
Actually the tool works with (knows about) many other ACS versions, not only those three.
Here is the complete list of bundled ACS inventories https://github.com/Alfresco/alfresco-extension-inspector/tree/master/extension-inspector-analyser/src/main/resources/bundled-inventories.

Reply:
You can rephrase above so it is clearer and concise :).

Reply:
Trying to resolve this

Reply:
Agreed with George/Denis the message should say 5.2 or later.